src/studio/src/designer/backend.Tests/Factories/ModelFactory/XsdToCSharpTests.cs

```diff
@@ -0,0 +1,131 @@
+using System;
+using System.IO;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+using System.Xml;
+using System.Xml.Serialization;
+using Altinn.Studio.Designer.Factories.ModelFactory;
+using Altinn.Studio.Designer.ModelMetadatalModels;
+using Designer.Tests.Utils;
+using Manatee.Json.Schema;
+using Xunit;
+
+namespace Designer.Tests.Factories.ModelFactory
+{
+    public class XsdToCSharpTests
+    {
+        [Fact]
+        public async void ConvertXsdToJsonSchema_CorrectXMLParsed()
+        {
+            // Arrange
+            using XmlReader xsdReader = XmlReader.Create(LoadTestData("Model/Xsd/Brønnøysundregistrene_ReelleRettighetshavere_M_2021-11-22_6900_46864_SERES.xsd"));
+            XsdToJsonSchema target = new XsdToJsonSchema(xsdReader);
+
+            // Act
+            JsonSchema xsdasJsonSchema = target.AsJsonSchema();
+
+            JsonSchemaToInstanceModelGenerator converter = new JsonSchemaToInstanceModelGenerator("test", "test", xsdasJsonSchema);
+            ModelMetadata modelMetadata = converter.GetModelMetadata();
+
+            string classes = GenerateCSharpClasses(modelMetadata);
+            Assembly assembly = Compiler.CompileToAssembly(classes);
+            Type type = assembly.GetType("Altinn.App.Models.ReelleRettighetshavere_M");
+            var modelInstance = assembly.CreateInstance(type.FullName);
```

Comment:
The initialized variable isn't being used. If the statement is needed you can discard the result.
```suggestion
            _ = assembly.CreateInstance(type.FullName);
```